src/sentry/static/sentry/app/actionCreators/group.jsx

```diff
@@ -17,16 +18,83 @@ export function assignTo(params) {
     // Sending an empty value to assignedTo is the same as "clear",
     // so if no member exists, that implies that we want to clear the
     // current assignee.
-    data: {assignedTo: (params.member && params.member.id) || ''},
+    data: {
+      assignedTo: params.user ? buildUserId(params.user.id) : '',
+    },
   });
 
   request
     .then(data => {
       GroupActions.assignToSuccess(id, params.id, data);
     })
     .catch(data => {
-      GroupActions.assignTodata(id, params.id, data);
+      GroupActions.assignToError(id, params.id, data);
     });
 
   return request;
 }
+
+export function clearAssignment(groupId) {
+  const api = new Client();
+
+  let endpoint = `/issues/${groupId}/`;
+
+  let id = api.uniqueId();
+
+  GroupActions.assignTo(id, groupId, {
+    email: '',
+  });
+
+  let request = api.requestPromise(endpoint, {
+    method: 'PUT',
+    // Sending an empty value to assignedTo is the same as "clear"
+    data: {
+      assignedTo: '',
+    },
+  });
+
+  request
+    .then(data => {
+      GroupActions.assignToSuccess(id, groupId, data);
+    })
+    .catch(data => {
+      GroupActions.assignToError(id, groupId, data);
+    });
+
+  return request;
+}
+
+export function assignToActor({id, actor}) {
+  const api = new Client();
+
+  let endpoint = `/issues/${id}/`;
+
+  let uniqueId = api.uniqueId();
```

Comment:
This doesn't do anything. It's passed to the `assignToSuccess` and `assignToError` functions but isn't used there